priv/templates/script.upload.eex

```diff
@@ -60,7 +60,20 @@ if [ -z "$FILENAME" ]; then
             # Try the pre-Nerves 1.4 path
             FIRMWARE_PATH="./_build/${MIX_TARGET}/${MIX_ENV}/nerves/images"
             if [ ! -d "$FIRMWARE_PATH" ]; then
-                echo "Can't find the build products. Specify path to .fw file or try running 'mix firmware'"
+                echo "Can't find the build products."
+                echo
+                echo "Nerves environment"
+                echo "MIX_TARGET:    ${MIX_TARGET}"
+                echo "MIX_ENV:       ${MIX_ENV}"
+                echo
+                echo "Make sure your Nerves environment is correct."
+                echo
+                echo "If the Nerves environment is correct make sure you have built the firmware using 'mix firmware'."
+                echo
+                echo "If you are uploading a .fw file from a custom path you can specify the path like so:"
+                echo
+                echo "  $0 device-destination /path/to/my/firmware.fw"
```

Comment:
```suggestion
                echo "  $0 <device hostname or IP address> </path/to/my/firmware.fw>"
```
I'm thinking that `device hostname or IP address` is more obvious than `device-destination`. The last change was to put brackets to show that you need to fill in these fields.

Also, have you tested this to see how everything looks after all of the changes? Like we didn't mess up line wraps or anything like that?